scripts/validateLinks.js

```diff
@@ -72,12 +72,15 @@ async function load_page_info(page, relative_path) {
     for (const match of content.matchAll(/ name="([^"]*)"/g)) {
         ret.fragments[match[1]] = true;
     }
+    for (const match of content.matchAll(/.{0,30}\]\[.{0,30}/g)) {
```

Comment:
Why the 0,30 constraint?

Reply:
prevent long text links? it is not obvious to me what this is doing either

Reply:
It's just capturing context to display in the error message, so I wanted to limit it to a reasonable size.